File: ad-hoc/python/3476.py
def calculate_sbc_time(S, B, C):
    rate_s = 1 / S
    rate_b = 1 / B
    rate_c = 1 / C
    
    rate_sbc = rate_s + rate_b + rate_c
    
    time_sbc = 1 / rate_sbc
    
    return time_sbc

S, B, C = map(int, input().split())

sbc_time = calculate_sbc_time(S, B, C)
print("{:.3f}".format(sbc_time))

File: ad-hoc/javascript/3476.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split(' ');

var S, B, C;

S = parseFloat(lines[0]);
B = parseFloat(lines[1]);
C = parseFloat(lines[2]);

var sbc_time = calculate_sbc_time(S, B, C);
console.log(`${sbc_time.toFixed(3)}`);
    

function calculate_sbc_time(S, B, C){
    var rate_s = 1 / S;
    var rate_b = 1 / B;
    var rate_c = 1 / C;
        
    var rate_sbc = rate_s + rate_b + rate_c;
        
    var time_sbc = 1 / rate_sbc;
        
    return time_sbc;
}